packages/zk-eddsa-event-ticket-pcd/src/CardBody.tsx

```diff
@@ -0,0 +1,97 @@
+import { FieldLabel, Separator, Spacer, TextContainer } from "@pcd/passport-ui";
+import styled from "styled-components";
+import { ZKEdDSAEventTicketPCD } from "./ZKEdDSAEventTicketPCD";
+
+export function ZKEdDSAEventTicketCardBody({
+  pcd
+}: {
+  pcd: ZKEdDSAEventTicketPCD;
+}) {
+  return (
+    <Container>
+      <p>
+        This PCD represents a response to a request for information about a
+        signed EdDSA ticket PCD that has been issued for a semaphore keypair
+        that the user possesses.
```

Comment:
maybe instead of 'represents a response to a request for information' we could communicate that this is a `EdDSA` ticket pcd with some of its fields obfuscated